pkg/server/handler.go

```diff
@@ -55,22 +55,22 @@ func (mc *MsgHandler) Stop() {
 
 func (mc *MsgHandler) subscribeToMessages(msgChan chan network.InMsg) error {
 	var err error
-	subscribe := func(msgChan chan network.InMsg, queueName, exchange, key string) {
+	subscribe := func(msgChan chan network.InMsg, queueName, exchange, kind, key string) {
 		if err != nil {
 			return
 		}
-		err = mc.amqp.OnMessage(msgChan, queueName, exchange, key)
+		err = mc.amqp.OnMessage(msgChan, queueName, exchange, kind, key)
 	}
 
 	// Subscribe to messages received from any client
-	subscribe(msgChan, queueNameFogIn, exchangeFogIn, bindingKeyDevice)
-	subscribe(msgChan, queueNameFogIn, exchangeFogIn, bindingKeySchema)
-	subscribe(msgChan, queueNameFogIn, exchangeFogIn, bindingKeyDeviceCommands)
-	subscribe(msgChan, queueNameFogIn, exchangeFogIn, bindingKeyPublishData)
+	subscribe(msgChan, queueNameFogIn, exchangeFogIn, "topic", bindingKeyDevice)
```

Comment:
Could this be a `const`? :thinking:
This way, if it ever changes, we only need to update it in a single place. In addition, its mode readable for the developer.
 
Also, same as [here](https://github.com/CESARBR/knot-babeltower/pull/66#discussion_r417625667), regarding `queueName...`

```Go
const exchangeTypeFogIn = "topic"
```

```suggestion
	subscribe(msgChan, queueNameFogIn, exchangeFogIn, exchangeTypeFogIn, bindingKeyDevice)
```

Reply:
Done.